adapters/adapters-commons/src/main/java/uk/gov/justice/raml/common/validator/RequestContentTypeRamlValidator.java

```diff
@@ -7,16 +7,12 @@
 import org.raml.model.MimeType;
 
 /**
- * Checks if action's mediaType is valid and contains a valid command or event.
+ * Checks if httpAction's mediaType is valid and contains a valid command or event.
  */
 public class RequestContentTypeRamlValidator extends AbstractContentTypeRamlValidator {
 
     public RequestContentTypeRamlValidator() {
-        super(ActionType.POST, "request type", "command", "event");
-    }
-
-    public RequestContentTypeRamlValidator(ActionType actionType, String contentTypeDec, String... componentTypes) {
-        super(actionType, contentTypeDec, componentTypes);
+        super(ActionType.POST, "request type");
```

Comment:
This 'magic string' "request type" seems fairly important, should it be elevated to a constant?